packages/toolbox/src/commands/chainlink/addAggregator.ts

```diff
@@ -79,9 +79,9 @@ export const handler = async (args: Options): Promise<void> => {
   }
   if (!aggregator.startsWith('0x')) {
     const aggregators = await getAllAggregators(network);
-    const newAggregator = aggregators.find((x) => x.pair === aggregator);
+    const newAggregator = aggregators.find((x) => x.name === aggregator);
     assert(newAggregator, `aggregator ${aggregator} not found`);
-    aggregator = newAggregator.proxy;
+    aggregator = newAggregator?.proxyAddress;
```

Comment:
I would specify aggregatorAddress or rename aggregators into aggregatorFeeds

Reply:
@olivier7delf I won't refactor the command in this PR but I'm creating a follow-up here: https://github.com/RequestNetwork/requestNetwork/pull/1195